src/components/tooltip/Marker.js

```diff
@@ -545,6 +545,17 @@ function getMarkerDetails(
         );
         break;
       }
+      case 'PreferenceRead': {
```

Comment:
same thing here: my understanding is that the `type` should be `Preference` :)

Also curiously this doesn't seem to work, see the tooltips in https://deploy-preview-2178--perf-html.netlify.com/public/53ce5ae957fdfeb423d466f435b5bb2e3555cdde/marker-chart/?globalTrackOrder=0-1-2-3-4-5-6&hiddenGlobalTracks=1-2-3-4-5&localTrackOrderByPid=26901-1-2-0~26929-0~27279-0~27073-0~27115-0~27019-0~26966-0-1~&markerSearch=preference&thread=7&v=4